variables.tf

```diff
@@ -56,6 +56,30 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "cloudwatch_policy_description" {
+  type        = string
+  description = "The description to associate with the IAM policy that allows read access to the specific CloudWatch log streams in which CDM is interested."
+  default     = "Allows read access to the specific CloudWatch log streams in which CDM is interested."
+}
+
+variable "cloudwatch_policy_instances" {
+  type        = list(string)
+  description = "Each string corresponds to the name of an instance (as it appears in the CloudWatch log group) for which CDM is to be allowed read access to the specific CloudWatch log groups in which they are interested."
```

Comment:
With the shift they're gaining access to specific streams now, right?

Reply:
Please have a look at the changes in e1df8a3 and let me know if they make things clearer.

Reply:
Oh yeah I understood what was happening, just the tail
> allowed read access to the specific CloudWatch log groups in which they are interested.

didn't seem to line up with how things worked now.